### PR TITLE
Relax missing label wording

### LINKED ISSUE #5368: Relax "missing label" wording in tooltips
I think this has been pointed out in some chat.

"app" and "version" labels are recommended in Istio https://istio.io/latest/docs/ops/deployment/requirements/#pod-requirements

but the "wording" of this tooltip can be relaxed, indicating that app/version labels are recommended as they affect telemetry; Istio routing can be performed through other custom labels, and perhaps the tooltip can bring some confusion to new users.

image

### PR DESCRIPTION
**Pull request by josunect**
Fixes #5368 

image

### CODE REVIEW

File: frontend/src/components/MissingLabel/MissingLabel.tsx [hunk 1]
```diff
@@ -40,7 +40,7 @@ class MissingLabel extends React.Component<MissingLabelProps, {}> {
               <PFBadge badge={{ badge: versionLabel }} isRead={true} style={{ marginRight: '0px' }} /> label is missing.{' '}
               <br />
             </div>
-            <div>This workload won't have Istio routing capabilities.</div>
+            <div>The label is recommended as it affects telemetry.</div>
           </>
         )}
         <div>Missing labels will impact in the telemetry collected by the Istio proxy.</div>
```
Comment (jshaughn) [inline frontend/src/components/MissingLabel/MissingLabel.tsx:46]:
```suggestion
        <div>Missing labels may impact telemetry reported by the Istio proxy.</div>
```

**Review by jshaughn — CHANGES_REQUESTED** [review]
@josunect The change is good, but let's also update that second line.  Thanks.

**Comment by josunect** [comment]
image

**Review by jshaughn — APPROVED** [review]
LGTM

**Pull request merged**